src/Cmp/Queues/Domain/Event/Subscriber.php

```diff
@@ -53,6 +59,8 @@ public function start($timeout=0)
         while(true) {
             try {
                 $this->queueReader->read(array($this, 'notify'), $timeout);
+            } catch(ParseMessageException $e){
```

Comment:
I'm not sure if the library should swallow poisoned tasks, I would like to know it instead.